Gems/HttpRequestor/Code/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ ly_add_target(
 # Load the "Gem::HttpRequestor" module in all types of applicatons.
 ly_create_alias(NAME HttpRequestor.Clients  NAMESPACE Gem TARGETS Gem::HttpRequestor)
 ly_create_alias(NAME HttpRequestor.Servers  NAMESPACE Gem TARGETS Gem::HttpRequestor)
+ly_create_alias(NAME HttpRequestor.Unified  NAMESPACE Gem TARGETS Gem::HttpRequestor)
```

Comment:
I wonder if it's worth creating a function to forward all known aliases for gems like these where every alias is the same e.g. 
```cmake
ly_create_default_alias(NAMESPACE Gem Targets Gem::HttpRequestor)
```

Reply:
I had this thought and it's hard to say where that would begin or end. There's quite a few aliases that can be specified. This change may begin a trend that makes a default set even less likely.